FT2023-269: Final commit for employee portal

ajay-mallah opened the pull request:
1. Added Employee content type.
2. Added Project content type.
3. Added user roles and permissions.
4. Added VIEWS

ajay-mallah committed:
FT2023-269: Final commit for employee portal

acheryyamanisha commented:
Please fix your module structure.
Your contrib modules are also added in the PR.

ajay-mallah committed:
FT2023-269: modified gitignore file.

ajay-mallah commented:
> Please fix your module structure. Your contrib modules are also added in the PR.

contrib modules  are added to the gitignore file and removed from the repository.
